src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const discount = 20;
+  const maxDiscount = 50;
+
+  if (days < 3) {
+    return (price * days);
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * price - discount);
+  }
```

Comment:
```suggestion
  if (days < 7) {
    return (days * price - discount);
  }
```
no need to check if days >=3 because it's obvious if the first check (days < 3) didn't pass.

Reply:
done!